### PR TITLE
Various types of cleaning up

### PR DESCRIPTION
**Pull request by hannesvdvreken**

### CODE REVIEW

File: src/Dispatcher.php [hunk 1]
```diff
@@ -116,10 +116,11 @@ protected function handleFound($handler, StrategyInterface $strategy, array $var
      * Handle a not found route
      *
      * @return \Symfony\Component\HttpFoundation\Response
+     * @throws \League\Route\Http\Exception\NotFoundException
      */
     protected function handleNotFound()
     {
-        $exception = new NotFoundException;
+        $exception = new NotFoundException();
```
Comment (philipobenito) [inline src/Dispatcher.php]: Parens can be removed

File: src/Strategy/RestfulStrategy.php [hunk 2]
```diff
@@ -18,7 +17,7 @@ public function dispatch($controller, array $vars)
         try {
             $response = $this->invokeController($controller, [
                 $this->getContainer()->get('Symfony\Component\HttpFoundation\Request'),
-                $vars
+                $vars,
```
Comment (philipobenito) [inline src/Strategy/RestfulStrategy.php]: I'd prefer no trailing comma

**Comment by hannesvdvreken** [comment]
@philipobenito Pushed new commit. If you want to merge ping me and give me a sec to rebase.

**Comment by philipobenito** [comment]
Ok for a rebase please @hannesvdvreken

**Comment by philipobenito** [comment]
Pulled patch in :+1:

**Comment by hannesvdvreken** [comment]
:+1:

**Pull request closed without merging**